src/bb-modules/System/Service.php

```diff
@@ -97,7 +97,7 @@ private function _getMultipleParams($params)
         $query = "SELECT param, value
                 FROM setting
                 WHERE param IN('".implode("', '", $params)."')
```

Comment:
Contains vulnrebility for a potential sql injection we need to filter those values before use in a query